src/Goerli-DssSpellCollateral.sol

```diff
@@ -28,61 +61,257 @@ contract DssSpellCollateralAction {
+    address constant RWA009_A_URN            = 0xD2C8588C72026171Ec3a17369ad0f0734E30915d;
+    address constant RWA009_A_JAR            = 0xa484C16D2Ca15706c4B875710d9e80b7F101572B;
+    // Goerli: CES Goerli Multisig / Mainnet: Genesis
+    address constant RWA009_A_OUTPUT_CONDUIT = 0x7a3D23Dc73F7ead55399597aAE6e525b3DF95A88;
+
+    // MIP21_LIQUIDATION_ORACLE params
+    string  constant RWA009_DOC              = "QmZG31b6iLGGCLGD7ZUn8EDkE9kANPVMcHzEYkvyNWCZpG";
+    uint256 constant RWA009_A_INITIAL_PRICE  = 100_000_000 * WAD; // No DssExecLib helper, so WAD is required
+    uint48  constant RWA009_A_TAU            = 0;
+
+    // Ilk registry params
+    uint256 constant RWA009_REG_CLASS_RWA    = 3;
+
+    // Remaining params
+    uint256 constant RWA009_A_LINE           = 100_000_000;
+    uint256 constant RWA009_A_MAT            = 100_00; // 100% in basis-points
+    uint256 constant RWA009_A_RATE           = ZERO_PCT_RATE;
+
+    // -- RWA009 END --
+
+    function onboardRwa008(
+        ChainlogAbstract CHANGELOG,
+        IlkRegistryAbstract REGISTRY,
+        address MIP21_LIQUIDATION_ORACLE,
+        address MCD_VAT,
+        address MCD_JUG,
+        address MCD_SPOT,
+        address MCD_JOIN_DAI
+    ) internal {
+        // RWA008-A collateral deploy
+        bytes32 ilk      = "RWA008-A";
+        uint256 decimals = DSTokenAbstract(RWA008).decimals();
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).vat() == MCD_VAT,  "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).ilk() == ilk,      "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).gem() == RWA008,   "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).dec() == decimals, "join-dec-not-match");
+
+        require(RwaUrnLike(RWA008_A_URN).vat() == MCD_VAT,               "urn-vat-not-match");
+        require(RwaUrnLike(RWA008_A_URN).jug() == MCD_JUG,               "urn-jug-not-match");
+        require(RwaUrnLike(RWA008_A_URN).daiJoin() == MCD_JOIN_DAI,      "urn-daijoin-not-match");
+        require(RwaUrnLike(RWA008_A_URN).gemJoin() == MCD_JOIN_RWA008_A, "urn-gemjoin-not-match");
+
+        // Init the RwaLiquidationOracle
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(ilk, RWA008_A_INITIAL_PRICE, RWA008_DOC, RWA008_A_TAU);
+        (, address pip, , ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Set price feed for RWA008
+        DssExecLib.setContract(MCD_SPOT, ilk, "pip", pip);
+
+        // Init RWA008 in Vat
+        Initializable(MCD_VAT).init(ilk);
+        // Init RWA008 in Jug
+        Initializable(MCD_JUG).init(ilk);
+
+        // Allow RWA008 Join to modify Vat registry
+        DssExecLib.authorize(MCD_VAT, MCD_JOIN_RWA008_A);
+
+        // Allow RwaLiquidationOracle2 to modify Vat registry
+        DssExecLib.authorize(MCD_VAT, MIP21_LIQUIDATION_ORACLE);
```

Comment:
We don't need this authorization as we went with the original oracle.

```suggestion
```

Reply:
Tests fail if this line is removed:
```
Failed tests:
[FAIL. Reason: Vat/not-authorized] testRWA008_INTEGRATION_TELL_CULL() (gas: 4415948)
[FAIL. Reason: Vat/not-authorized] testRWA009_INTEGRATION_TELL_CULL() (gas: 4411903)
```

@gbalabasquer thinks that maybe this step was not executed when MCD was migrated from Kovan to Goerli.